src/styles/variables.scss

```diff
@@ -0,0 +1,15 @@
+$widthOfDay: 100px;
+$heigthOfDay: $widthOfDay;
```

Comment:
There is a typo in the variable name `$heigthOfDay`. It should be `$heightOfDay` to correctly represent the word 'height'.